packages/textlint/src/cli.ts

```diff
@@ -76,11 +76,26 @@ export const cli = {
             const descriptor = await loadDescriptor(currentOptions);
             Logger.log(JSON.stringify(descriptor, null, 4));
             return Promise.resolve(0);
+        } else if (currentOptions.mcp) {
+            const { server: mcpServer } = require("@textlint/mcp");
+            const { StdioServerTransport } = require("@modelcontextprotocol/sdk/server/stdio.js");
+
+            await mcpServer.connect(new StdioServerTransport());
+            // NOTE: do not use Logger.log() because stdout use to the server transport
+            Logger.error(`Textlint MCP server is running.`);
```

Comment:
Question: Is it possible to use Logger.log before connecting?

```suggestion
            // NOTE: do not use Logger.log() after connect because stdout use to the server transport
            Logger.log(`Textlint MCP server is running.`);
            await mcpServer.connect(new StdioServerTransport());
```

Reply:
Thanks.
Even after moving the log output before the connection, running `npx @modelcontextprotocol/inspector npx textlint --mcp` still resulted in the following error:
```
Stdio transport: command=/Users/chick-p/.local/share/mise/installs/node/18.20.1/bin/npx, args=textlint,--mcp
Spawned stdio transport
Connected MCP client to backing server transport
Created client/server transports
Set up MCP proxy
Received message for sessionId 55d77644-d73a-4d64-9187-89924c7ebd0f
Error from MCP server: SyntaxError: Unexpected token T in JSON at position 0
    at JSON.parse (<anonymous>)
    at deserializeMessage (file:///Users/chick-p/.npm/_npx/5a9d879542beca3a/node_modules/@modelcontextprotocol/sdk/dist/esm/shared/stdio.js:26:44)
    at ReadBuffer.readMessage (file:///Users/chick-p/.npm/_npx/5a9d879542beca3a/node_modules/@modelcontextprotocol/sdk/dist/esm/shared/stdio.js:19:16)
    at StdioClientTransport.processReadBuffer (file:///Users/chick-p/.npm/_npx/5a9d879542beca3a/node_modules/@modelcontextprotocol/sdk/dist/esm/client/stdio.js:127:50)
    at Socket.<anonymous> (file:///Users/chick-p/.npm/_npx/5a9d879542beca3a/node_modules/@modelcontextprotocol/sdk/dist/esm/client/stdio.js:98:22)
    at Socket.emit (node:events:517:28)
    at addChunk (node:internal/streams/readable:368:12)
    at readableAddChunk (node:internal/streams/readable:341:9)
    at Readable.push (node:internal/streams/readable:278:10)
    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)
Received message for sessionId 55d77644-d73a-4d64-9187-89924c7ebd0f
Received message for sessionId 55d77644-d73a-4d64-9187-89924c7ebd0f
Received message for sessionId 55d77644-d73a-4d64-9187-89924c7ebd0f
```

In https://github.com/textlint/textlint/pull/1522, I tried removing the log output entirely. What do you think about this approach?